README.md

```diff
@@ -1,9 +1,9 @@
 # chapel-vscode extension
 
-## Features
+## Core Features
 
 - Intellisense
-  - GotoDefinition: jump to a symbols definition
+  - GotoDefinition: jump to a symbol's definition
   - Hover: hover over a symbol to view its signature and documentation
   - Code Completion: suggest completion for symbols in the current file
   - Symbol List: see all symbols for a file
```

Comment:
We also have the ability to jump to or peek all references for a certain symbol right? I think Daniel had demoed that at some point. We should add that in here too.

Reply:
Done!